hotdoc/extensions/gst/gst_extension.py

```diff
@@ -831,7 +836,7 @@ def __create_enum_symbol(self, type_name, enum, element_name, parent_name=None):
         return symbol
 
     def __create_object_type(self, pagename, _object):
-        if not _object:
+        if not _object or "hierarchy" not in _object:
```

Comment:
when can that happen?

Reply:
When it is a GstObject/GstElement as those are skipped in the cache.

Reply:
so the whole "let's mark things as plugin API" discussion / effort started because of this discussion, is this still needed? :)

Reply:
Forgot to remove it all :)